classes/ValidFormBuilder/MultiField.php

```diff
@@ -147,6 +148,8 @@ public function __construct($label, $meta = array())
 
         $this->__dynamic = $this->getMeta("dynamic", $this->__dynamic);
         $this->__dynamicLabel = $this->getMeta("dynamicLabel", $this->__dynamicLabel);
+
+        $this->initialiseDynamicRemoveMeta();
```

Comment:
Yeah unfortunately I must agree. At first it started out as a great plan but later on when the implementation started to become more complex, stuff like this line happened. 

Yesterday I figured this would need a rewrite where the Trait would be replaced. I was thinking about creating a `RemovableElement` class which extends the `Element` class. 

Perhaps the best solution is to simply hardcode the removable logic in the code base. Dynamic fields can be added and removed, period.